ci/install-build-deps.sh

```diff
@@ -6,22 +6,25 @@
 # modify the contents of /etc/apt. The script is intended to be run inside a container of the corresponding OS, not directly on your machine.
```

Comment:
Can we add modify the contents of /etc/apt and /etc/os-release here.

Reply:
`/etc/os-release` is not modified.